pages/index.js

```diff
@@ -6,7 +6,7 @@ import Community from '../components/RootHomePage/Community';
 import Events from '../components/RootHomePage/Events';
 import Hero from '../components/RootHomePage/Hero';
 import Keynotes from '../components/RootHomePage/Keynotes';
-import LiveStream from '../components/RootHomePage/LiveStream';
+// import LiveStream from '../components/RootHomePage/LiveStream';
```

Comment:
Was this giving you issues when you pulled in my branch?

Reply:
Nope commented it out as part of figuring out why there was horizontal scroll on mobile and forgot to uncomment it.